examples/angularjs/js/services/todoStorage.js

```diff
@@ -34,18 +34,12 @@ angular.module('todomvc')
 			),
 
 			clearCompleted: function () {
-				var originalTodos = store.todos.slice(0);
-
-				var incompleteTodos = store.todos.filter(function (todo) {
-					return !todo.completed;
+				var completeTodos = store.todos.filter(function (todo) {
+					return todo.completed;
 				});
-
-				angular.copy(incompleteTodos, store.todos);
-
-				return store.api.delete(function () {
-					}, function error() {
-						angular.copy(originalTodos, store.todos);
-					});
+				for (var i = 0; i < completeTodos.length; i++) {
+					store.delete(completeTodos[i]);
+				};
```

Comment:
No need to have a semicolon here. Kinda surprises me that the linter doesn't check for this? :/